src/types/common.rs

```diff
@@ -769,6 +769,17 @@ pub struct FolderId {
     pub change_key: Option<String>,
 }
 
+/// The element that determines how elements are deleted.
```

Comment:
```suggestion
/// The manner in which items or folders are deleted.
```

suggestion: This is a little bit more explicit about what it affects.